packages/app-staking/src/types.ts

```diff
@@ -14,11 +14,10 @@ export type Nominators = {
 
 export type ComponentProps = {
   allAccounts?: SubjectInfo,
-  balances?: DerivedBalancesMap,
-  controllers: Array<string>,
-  recentlyOffline: RecentlyOfflineMap,
-  stashes: Array<string>,
-  validators: Array<string>
+  allControllers: Array<string>,
```

Comment:
Just a comment, when we swap to eslint, the typescript-recommended rules (which we use as-is, less bike-shedding all-around), wants Arrays as `string[]` - would be good if we start using that in new code already and get used to it. (Plus, also less to adjust with the eslint PR that is ongoing)